couchbase/bucket.hxx

```diff
@@ -213,7 +214,7 @@ class bucket : public std::enable_shared_from_this<bucket>
         } else {
             new_session = std::make_shared<io::mcbp_session>(client_id_, ctx_, origin_, name_, known_features_);
         }
-        new_session->bootstrap([self = shared_from_this(), new_session, h = std::forward<Handler>(handler)](
+        new_session->bootstrap([self = shared_from_this(), new_session = std::move(new_session), h = std::forward<Handler>(handler)](
```

Comment:
yes, it is not necessary here, but above it is the last usage of the pointer, so we can safely move it into the closure